assets/src/domain/blocks/event-attendees/controls/SelectOrderBy.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { SelectControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+import { AttendeesEditProps } from '../types';
+
+const SelectOrderBy: React.FC<AttendeesEditProps> = ({ attributes, setAttributes }) => {
```

Comment:
this could also be extracted into something like:

```
assets/src/domain/blocks/components/OrderByControl
```

or

```
assets/src/domain/blocks/components/OrderBySelect
```

or

```
assets/src/domain/blocks/components/OrderBySelector
```

but will obviously require the options to be supplied as a prop